region/wreckedship/main.json

```diff
@@ -1632,7 +2049,40 @@
           "to": [
             {
               "id": 2,
-              "requires": null
+              "strats": [
```

Comment:
Springball bounce on the first and third platforms

Reply:
Neat